modules/admin_manual/pages/enterprise/external_storage/windows-network-drive_configuration.adoc

```diff
@@ -138,49 +130,115 @@ The ownCloud windows network drive listener utilizes the SMB notification featur
-+
-.WND mountpoint and authorization credentials
-image::enterprise/external_storage/windows_network_drive/wnd-2.png[WND mountpoint and authorization credentials, width=500]
+
 . Enter the address of the server that contains the WND share.
-. The share name provided by Windows or Samba.
-. The root folder of the share. This is can be a subfolder name, or the `$user` variable for the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
+
+. Use the share name provided by Windows or Samba.
+
+. Optionally, the root folder of the share. This can be a subfolder name, or the `$user` variable when assigning the user's home directory. Note that the LDAP `Internal Username Attribute` must be set to the `samaccountname` for either the share or the root to work, and the user's home directory needs to match the `samaccountname`.
 (See xref:configuration/user/user_auth_ldap.adoc[User Authentication with LDAP].)
-. Login credentials.
-. Select users or groups with access to the share. The default is all users.
-. Click the gear icon for additional mount options. Note that previews are enabled by default, while
-sharing is not (see Figure 3). Sharing is not available for all authorization methods. For details please see the
-xref:enterprise/external_storage/enterprise_only_auth.adoc#authentication-option-details[Enterprise-Only Authentication Options]. When using large storages with many files, you may want to disable previews, because this can significantly increase performance.
+
+. Add the login credentials, if required, depending on the authentication method selected.
+
+. Sharing
+
+.. Admin only menu:Settings[Admin > Storage]: +
+Admins can define that mount points can be shared to all (default), or be restricted to individuals or groups. Note that sharing is not available for all authorization methods. For details please see the
+xref:enterprise/external_storage/enterprise_only_auth.adoc#authentication-option-details[Enterprise-Only Authentication Options]. In this case, you will also see the following on the mount point: 
++
+image::enterprise/external_storage/windows_network_drive/wnd-available-for.png[WND Sharing Options,width=400]
+
+.. All menu:Settings[General > Storage]: +
+If sharing of mounts is allowed by the admin via menu:Settings[Admin > Storage > Allow sharing on user-mounted external storages], users can share the mount via menu:Files[Mountpoint > Details > Sharing]. To avoid accidentially sharing resources, the user must allow sharing a mount in the mount points view upfront by clicking on the gear icon as shown below. btn:[Enable sharing] appears only if sharing is generally allowed by the admin, see above.
```

Comment:
```suggestion
If sharing of mounts is allowed by the admin via menu:Settings[Admin > Storage > Allow sharing on user-mounted external storages], users can share the mount via menu:Files[Mountpoint > Details > Sharing]. To avoid accidentally sharing resources, the user must allow sharing a mount in the mount points view upfront by clicking on the gear icon as shown below. btn:[Enable sharing] appears only if sharing is generally allowed by the admin, see above.
```